### PR TITLE
feat: LT-1233 - WMD end use concerns section

### PR DESCRIPTION
**Pull request by andytaylorkainos**

### CODE REVIEW

File: ui_automation_tests/step_defs/test_organisation.py [hunk 1]
```diff
@@ -20,10 +20,10 @@ def verify_registered_organisation(driver, context):
     assert functions.element_with_css_selector_exists(driver, ".lite-info-bar")
     OrganisationsPage(driver).search_for_org_in_filter(context.organisation_name)
     row = OrganisationPage(driver).get_organisation_row()
-    assert context.organisation_name == row["name"]
-    assert context.eori == row["eori-number"]
-    assert context.sic == row["sic-number"]
-    assert context.vat == row["vat-number"]
+    assert context.organisation_name in row["name"]
```
Comment (stephanieboyd) [inline ui_automation_tests/step_defs/test_organisation.py]: Why this change?

Comment (andytaylorkainos) [reply ui_automation_tests/step_defs/test_organisation.py]: tests broke somehow. the text being returned included the title:
"Name: Organisation X" etc
